src/main/java/com/amazon/opendistroforelasticsearch/jobscheduler/scheduler/JobScheduler.java

```diff
@@ -164,7 +166,11 @@ boolean reschedule(ScheduledJobParameter jobParameter, JobSchedulingInfo jobInfo
             }
         }
 
-        jobInfo.setExpectedExecutionTime(nextExecutionTime);
+        // Add the defined delay to the job execution
+        if (jobParameter.getDelaySeconds() != null) {
+            log.info("Delay: {} seconds", jobParameter.getDelaySeconds());
```

Comment:
This will print a too much delay log. How about we set this log at `debug` level and add more information like job name?

Reply:
Thanks for pointing out. I forgot to polish the log message before pushing the codes. I with change the log level and the message content in tomorrow.

Reply:
Have updated the log level and message.